### DISCUSSION TITLE
enabling multipart resulted in missing package

### DISCUSSION THREAD

**Discussion by shi-yan** [tokio-rs/axum#2900] [Q&A] [answered]
### Summary

I tried to use the multipart feature:

```toml
axum = { version = "0.7", features = ["multipart"]}
```
which results in the following compiling error:
```bash
error: failed to select a version for the requirement `multer = "^3.0.0"`
candidate versions found which didn't match: 2.1.0, 2.0.4, 2.0.3, ...
location searched: crates.io index
required by package `axum v0.7.5`
```
the latest version of multer seems to be 2.x, where to find 3.x?

### axum version

0.7.5

**Comment by jplatte** [comment, marked as answer]
> the latest version of multer seems to be 2.x

I'm seeing 3.1.0 on https://crates.io/crates/multer.

**Reply by shi-yan** [reply]
yes, somehow if I manually add multer="3.1" as my dependency, it worked. but only adding axum with the multipart feature doesn't
